core/src/main/scala/kafka/admin/ReassignPartitionsCommand.scala

```diff
@@ -812,6 +812,26 @@ object ReassignPartitionsCommand extends Logging {
     proposedAssignments
   }
 
+  private def describeTopics(adminClient: Admin,
+                             topics: Set[String])
+                             : Map[String, TopicDescription] = {
+    adminClient.describeTopics(topics.asJava).values.asScala.map {
+      case (topicName, topicDescriptionFuture) =>
+        try {
+          topicName -> topicDescriptionFuture.get
+        }
+        catch {
+          case t: ExecutionException if t.getCause != null =>
```

Comment:
How about using `classOf[UnknownTopicOrPartitionException].isInstance(t.getCause)`? That includes null check.